docs/usage.rst

```diff
@@ -0,0 +1,98 @@
+*****
+Usage
+*****
+
+Run Hyperparameter Search
+=========================
+
+cluster_utils provides two main commands to run hyperparameter search in different
+modes:
```

Comment:
```suggestion
cluster_utils provides two main commands to interface with the cluster:
```

In my view, "hyperparameter search" is not the primary description of what cluster_utils does. It is the major usecase, but fundamentally I see cluster_utils as providing a convenient and unified interface to the cluster.

Reply:
I was actually a bit unsure if "hyperparameter search" is a good overarching term, so I'm happy to change it.  Just "interface with the cluster" seems a bit too vague to me, though.  How about "to run batches of jobs on the cluster" (picking up the nomenclature from the headline)?